rules/sinks/third_parties/sdk/google/kotlin.yaml

```diff
@@ -7,4 +7,21 @@ sinks:
       - "(?i)(com[.]google[.]firebase)[.](?!(database)).*"
       - "(?i)(com[.]firebase)[.](?!(database)).*"
       - "(?i)(io[.]firebase)[.](?!(database)).*"
+      - "(?i)(com[.]google[.]firebase[.]remoteconfig).*"
+    tags:
+
+  - id: ThirdParties.SDK.Google.Firebase.Crashlytics
+    name: Firebase
```

Comment:
Please update name and domain as per other languages for consistency
https://github.com/Privado-Inc/privado/blob/acd0e37740c98a4e8b38da5aebb72c7c539dd260/rules/sinks/third_parties/sdk/google/java.yaml#L1001

Reply:
Thanks @HiteshMah-Jan for pointing this out.